MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -112,7 +107,17 @@ public string DisplayTitle
                         attributes.Add("Default");
                     }
 
-                    return string.Join(" ", attributes);
+                    if (!string.IsNullOrEmpty(Title))
+                    {
+                        return attributes
+                            // keep Tags that are not already in Title
+                            .Where(tag => Title.IndexOf(tag, StringComparison.OrdinalIgnoreCase) == -1)
```

Comment:
I wonder if this check is too broad, especially for the language attribute. What if the language tag is 'Eng' as in your example, and the title contains the word 'Revenge' or 'Challenge'?

Reply:
That's true. I was just pulling based on the check done for subtitles. Perhaps it should only check for complete equality. So if someone titles the audio track as Eng, it would only display `Eng` instead of `Eng - Eng`. I would do a starts with check, but with language tags only being 3 letters long for most audio tracks, it may make sense to only check exact equality. Thoughts?

Reply:
Opted to instead convert the language tag to the full language name, and keep the filter as is, so words like `Challenge` or `Revenge` won't be an issue.